src/query/spec.ts

```diff
@@ -61,7 +61,9 @@ export function fromSpec(spec: ExtendedUnitSpec): SpecQuery {
 
 export function isAggregate(specQ: SpecQuery) {
   return some(specQ.encodings, (encQ: EncodingQuery) => {
-    return (!isWildcard(encQ.aggregate) && !!encQ.aggregate) || encQ.autoCount === true;
+    // TODO(akshatsh): aggregates are only fieldQueries?
```

Comment:
Yes.  (But it will become either FieldQuery or AutoCountQuery once you add the latter.)